docs/markdown/spline_intro.md

```diff
@@ -208,6 +218,15 @@ The NURBS basis functions :math:`R_{i,p}` are computed using the B-spline basis
 
 Again, :math:`\omega_i` is a weight.
 
+
+#### Properties of NURBS basis functions:
+
+The same properties of B-spline basis functions hold with the following additions:
+
+- computation of a set of basis functions additionally requires the weights :math:`\{\omega_i\}`
+- each basis function attains exactly one maximum in the interval :math:`[0,1]`, except for the case :math:`p=0`
```

Comment:
shouldn't `[0, 1]` be the same as above?

Reply:
I think so. I'll correct it